src/Instrumentation/MySqli/src/MySqliInstrumentation.php

```diff
@@ -440,11 +448,52 @@ private static function constructPostHook(int $paramsOffset, CachedInstrumentati
     }
 
     /** @param non-empty-string $spanName */
-    private static function queryPreHook(string $spanName, CachedInstrumentation $instrumentation, MySqliTracker $tracker, $obj, array $params, ?string $class, string $function, ?string $filename, ?int $lineno): void
+    private static function queryPreHook(string $spanName, ?TextMapPropagatorInterface $contextPropagator, CachedInstrumentation $instrumentation, MySqliTracker $tracker, $obj, array $params, ?string $class, string $function, ?string $filename, ?int $lineno): array
     {
         $span = self::startSpan($spanName, $instrumentation, $class, $function, $filename, $lineno, []);
         $mysqli = $obj ? $obj : $params[0];
+        $query = $obj ? $params[0] : $params[1];
+        $query = mb_convert_encoding($query ?? self::UNDEFINED, 'UTF-8');
+        if (!is_string($query)) {
+            $query = self::UNDEFINED;
+        }
+        $span->setAttributes([
+            TraceAttributes::DB_QUERY_TEXT => $query,
+            TraceAttributes::DB_OPERATION_NAME => self::extractQueryCommand($query),
+        ]);
+
         self::addTransactionLink($tracker, $span, $mysqli);
+
+        if (class_exists('OpenTelemetry\Contrib\SqlCommenter\SqlCommenter') && $query !== self::UNDEFINED) {
```

Comment:
In the previous PR #406 the SQL Commenter feature was enabled via an explicit env var config. This PR proposes opting-in by includinging the package instead, correct? Can the READMEs for each database instrumentation be updated to provide usage / configuration info?